bot/extensions/levelling/commands.py

```diff
@@ -472,6 +484,67 @@ async def list_roles(self, interaction: core.InteractionType):
 
         return await interaction.response.send_message(f"```\n{response}\n```\n")
 
+    @config.command(name="xp_boost")
+    async def xp_boost(self, interaction: discord.Interaction, multiplier: int):
+        """Change the xp multiplier"""
+        if 0 < multiplier <= 10:
+            self.xp_boost = multiplier
+            query = """
+                    UPDATE configs
+                       SET xp_boost = $1
+                     WHERE guild_id = $2
+                    """
+            await Config.execute(query, multiplier, interaction.guild.id)
+            return await interaction.response.send_message(f"XP multiplied by {multiplier}x.")
+        else:
+            return await interaction.response.send_message("XP multiply range should be between 1 and 10")
```

Comment:
I don't think we've discussed a limit for the range, but I do agreed that there should be one:)
My only comment here is that you should have used a guard clause instead of this else: